frontend/internal-packages/agent/src/chat/workflow/README.md

```
@@ -8,22 +8,19 @@ A **LangGraph implementation** for processing chat messages in the LIAM applicat
 %%{init: {'flowchart': {'curve': 'linear'}}}%%
 graph TD;
 	__start__([<p>__start__</p>]):::first
-	analyzeRequirements(analyzeRequirements)
-	invokeSaveArtifactTool(invokeSaveArtifactTool)
+	pmAgent(pmAgent)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)